### PR TITLE
AP_Airspeed: Fix ARSPD2_TYPE docs

### PR DESCRIPTION
**Pull request by DavidIngraham**
Updated to match ARSPD_TYPE

### CODE REVIEW

File: libraries/AP_Airspeed/AP_Airspeed.cpp [hunk 1]
```diff
@@ -128,7 +128,7 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Param: 2_TYPE
     // @DisplayName: Second Airspeed type
     // @Description: Type of 2nd airspeed sensor
-    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-SDP3X
+    // @Values: 0:None,1:I2C-MS4525D0,2:Analog,3:I2C-MS5525,4:I2C-MS5525 (0x76),5:I2C-MS5525 (0x77),6:I2C-SDP3X
```
Comment (magicrub) [inline libraries/AP_Airspeed/AP_Airspeed.cpp]: do we want index 4 to say AUTO?

Comment (DavidIngraham) [reply libraries/AP_Airspeed/AP_Airspeed.cpp]: I assume you mean option 3? We can definitely do that. We would want to change the doc for ARSPD_TYPE as well.

**Comment by magicrub** [comment]
I think there was already some discussion about it with @WickedShell

**Comment by magicrub** [comment]
@DavidIngraham looks like you've pushed this branch to upstream.

**Comment by DavidIngraham** [comment]
Yep, that was a mistake (I was using the web interface and I thought it was going to make the new branch on my fork). My apologies.

Would it be best to close this, or just delete it after merging?

**Comment by magicrub** [comment]
merged and I deleted the branch since this at least brings it inline with the existing _TYPE param. We can discuss adding "auto" to both TYPE and TYPE2 later

**Comment by DavidIngraham** [comment]
Cool, Thanks Tom!

**Pull request merged**